pkg/dependency/parser/python/uv/parse_testcase.go

```diff
@@ -9,23 +9,44 @@ var (
 	// uv init normal && cd normal
 	// uv add requests==2.32.0
 	// uv add --group dev pytest==8.3.4
+	// uv add httpx==0.28.1 --extra socks
+	// uv add orjson==3.10.12 --optional json
 	// apk add jq
 	// uv pip list --format json |jq -c 'sort_by(.name) | .[] | {"ID": (.name + "@" + .version), "Name": .name, "Version": .version}' | sed 's/$/,/' | sed 's/\"\([^"]*\)\":/\1:/g'
 
 	// add a root project
 	// fill in the relationships between the packages
 	uvNormal = []ftypes.Package{
 		{ID: "normal@0.1.0", Name: "normal", Version: "0.1.0", Relationship: ftypes.RelationshipRoot},
+		{ID: "httpx@0.28.1", Name: "httpx", Version: "0.28.1", Relationship: ftypes.RelationshipDirect},
```

Comment:
This package is not marked as a development dependency. Is it correct?

Reply:
`httpx` is not a development dependency: `uv add httpx==0.28.1 --extra socks`

Reply:
Why did we newly introduce this dependency? I thought the test case was updated for optional or development dependencies.

Reply:
Did we need it to test extra packages?

Reply:
Ah, I should have added about test cases in the description. Yes, I added some more test cases:
- An optional dependency in the root package
- Direct dependency with an extra dependency that is optional

Reply:
OK, thanks.